src/domain/app/layout/PageLayout.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FunctionComponent } from 'react';
 import { HelmetProvider } from 'react-helmet-async';
 
 import Header from '../header/Header';
-import styles from './layout.module.scss';
+import styles from './pageLayout.module.scss';
```

Comment:
without this renaming
in sass file 
`@import 'layout'` will import the tsx file lol

Reply:
@Zeukkari